autoload/vader/window.vim

```diff
@@ -38,13 +38,16 @@ function! s:switch_to_workbench()
   execute 'b' s:workbench_bfr
 endfunction
 
-function! vader#window#open()
+function! vader#window#close()
   silent! bd \[Vader\]
-  silent! bd \[Vader-workbench\]
+  silent! bw! \[Vader-workbench\]
```

Comment:
Any reason you changed `bd` to `bw!` here, while leaving line 42 unchanged? And can you explain why the change was necessary? Did you have any issues with `bd`?